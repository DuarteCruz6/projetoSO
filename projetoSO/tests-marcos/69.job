WRITE [(k2,7)(u4,66)(m3,75)(f4,42)(v7,18)]
WRITE [(i2,7)]
DELETE [k2,f4,m3,i2]
WRITE [(j0,91)(k3,30)(x8,60)]
WRITE [(x4,48)(h4,18)(s4,22)(z0,35)(a6,40)]
BACKUP
WRITE [(c7,76)(p9,55)(t9,2)(g0,79)(i7,59)]
READ [u4]
WRITE [(g3,89)(a1,25)(u0,42)(w1,70)]
DELETE [a1,k3,s4]
DELETE [u4,w1]
READ [i7]
SHOW
READ [z0]
READ [h4]
WRITE [(o0,49)]
WRITE [(n1,47)(i6,88)(o4,51)(e2,66)]
BACKUP
WRITE [(t3,87)(c3,52)]
WRITE [(p2,0)(i3,70)(s4,82)(r3,90)]
WRITE [(j5,65)(b2,10)(z4,44)(b5,59)]
READ [p2]
WRITE [(r5,61)(c9,62)(r7,7)(e5,91)(t4,47)]
WRITE [(y2,57)(x7,19)]
READ [b2]
DELETE [r3,y2,e2]
READ [b5]
WRITE [(h9,48)]
DELETE [i3,c3,i7]
WRITE [(o5,88)(m8,87)]
READ [b5]
WRITE [(n9,40)(f4,34)(n3,90)(u1,71)(l9,49)]
DELETE [r5,o4,t9,g3,n9]
READ [x4]
DELETE [g0,a6,b2,z4,t3]
WRITE [(j9,99)(o7,48)(l7,51)(m2,26)]
DELETE [j5,e5,j0,l7]
WRITE [(b6,95)]
WRITE [(e5,60)]
WRITE [(q3,39)]
WRITE [(d3,67)(p6,15)(a9,61)(n0,41)]
READ [p6]
WRITE [(y1,16)(u5,46)(y4,65)(f6,27)]
READ [u5]
WRITE [(j6,72)]
