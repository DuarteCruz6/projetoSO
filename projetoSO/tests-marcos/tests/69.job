WRITE [(i1,52)]
WRITE [(x9,41)(z8,28)(k1,3)(o1,20)(m5,56)]
WRITE [(w8,57)(b2,68)(p2,10)]
WRITE [(n9,86)(s6,5)]
WRITE [(b9,4)]
WRITE [(x2,69)(m3,74)(d7,32)(t8,74)(a1,46)]
WRITE [(l9,78)(e8,67)]
WRITE [(s5,93)(h5,1)(a5,44)(y2,91)]
DELETE [m3,y2,i1,a5,x9]
BACKUP
SHOW
WAIT 2000
WRITE [(w1,58)(u0,5)(q3,82)]
WRITE [(d5,60)(l7,14)(b1,97)(i1,51)]
WRITE [(s1,96)(r5,67)(n5,11)]
WRITE [(t4,13)(t6,77)(m2,52)(c1,16)(j6,74)]
READ [x2]
WAIT 3000
WRITE [(f7,65)(i6,80)(m8,67)(h9,0)]
READ [h5]
SHOW
WRITE [(h7,59)(u6,46)]
WAIT 2000
WAIT 3000
WRITE [(h6,72)(a9,98)]
WRITE [(m4,34)(z7,60)]
DELETE [t8,h9,p2,h7]
WRITE [(j3,89)(x4,86)(i9,39)(r7,23)]
READ [o1]
WRITE [(x8,8)(t2,89)(b8,51)]
WRITE [(s8,68)(s7,69)(e6,78)(h7,49)]
WRITE [(z5,49)]
DELETE [n9,e6,n5]
READ [w1]
DELETE [w1,j3,z8,e8]
BACKUP
WRITE [(j4,37)(y0,82)(u3,84)(s0,89)(x9,36)]
WRITE [(v4,60)(s9,96)]
WRITE [(r6,37)(c4,22)(k6,9)]
WRITE [(g9,16)(p0,46)]
WRITE [(b4,60)(j0,74)(z4,10)(l8,7)(v0,26)]
WRITE [(f0,26)]
WRITE [(n9,35)]
DELETE [k1,s5,m2,r5,t4]
