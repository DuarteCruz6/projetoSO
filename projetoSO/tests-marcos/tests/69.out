(a1, 46)
(b9, 4)
(b2, 68)
(d7, 32)
(e8, 67)
(h5, 1)
(k1, 3)
(l9, 78)
(m5, 56)
(n9, 86)
(o1, 20)
(p2, 10)
(s5, 93)
(s6, 5)
(t8, 74)
(w8, 57)
(x2, 69)
(z8, 28)
[(x2,69)]
[(h5,1)]
(a1, 46)
(b1, 97)
(b9, 4)
(b2, 68)
(c1, 16)
(d5, 60)
(d7, 32)
(e8, 67)
(f7, 65)
(h9, 0)
(h5, 1)
(i6, 80)
(i1, 51)
(j6, 74)
(k1, 3)
(l7, 14)
(l9, 78)
(m8, 67)
(m2, 52)
(m5, 56)
(n5, 11)
(n9, 86)
(o1, 20)
(p2, 10)
(q3, 82)
(r5, 67)
(s1, 96)
(s5, 93)
(s6, 5)
(t6, 77)
(t4, 13)
(t8, 74)
(u0, 5)
(w1, 58)
(w8, 57)
(x2, 69)
(z8, 28)
[(o1,20)]
[(w1,58)]
